File: Partner.Eligibility.REST.VB/IEligibilityService.vb
Imports System.IO
Imports System.ServiceModel
Imports System.ServiceModel.Web

<ServiceContract()>
Public Interface IEligibilityService
	<OperationContract()>
	<WebInvoke(BodyStyle:=WebMessageBodyStyle.Wrapped, Method:="GET", ResponseFormat:=WebMessageFormat.Xml, UriTemplate:="/getmemberinfo?memberId={memberId}")>
	Function GetMemberInfo(ByVal memberId As String) As Stream
End Interface

File: Partner.Eligibility.REST/IEligibilityService.cs
using System.IO;
using System.ServiceModel;
using System.ServiceModel.Web;

namespace Partner.Eligibility.REST
{
	[ServiceContract]
	public interface IEligibilityService
	{
		[OperationContract]
		[WebInvoke(BodyStyle = WebMessageBodyStyle.Wrapped, Method = "GET", ResponseFormat = WebMessageFormat.Xml, UriTemplate = "/getmemberinfo?memberId={memberId}")]
		Stream GetMemberInfo(string memberId);
	}
}
